salt/states/pip_state.py

```diff
@@ -163,8 +164,11 @@ def _check_pkg_version_format(pkg):
         ret['version_spec'] = []
     else:
         ret['result'] = True
-        ret['prefix'] = install_req.req.project_name
-        ret['version_spec'] = install_req.req.specs
+        if hasattr(install_req.req, 'project_name'):
```

Comment:
``install_req.name`` not ``install_req.req.name``. Notice the missing extra ``.req``.